sdks/go/pkg/beam/options/gcpopts/options.go

```diff
@@ -37,9 +39,22 @@ var (
 // GetProject returns the project, if non empty and exits otherwise.
 // Convenience function.
 func GetProject(ctx context.Context) string {
-	if *Project == "" {
+	project := GetProjectFromFlagOrEnvironment(ctx)
```

Comment:
While we can't meaningfully test the "exit" path, can we consider adding a test for the GetProject happy path (where the project is set)? That will probably satisfy the code coverage diff, and overall increase coverage on this file.

Reply:
Good call - I'd initially missed this and then caught that this function needed updated as well as I was self code-reviewing, clearly missed the test though!